iceberg/iceberg-handler/src/test/results/positive/iceberg_stats.q.out

```diff
@@ -8,6 +8,16 @@ POSTHOOK: query: create table ice01 (id int, key int) Stored by Iceberg stored a
 POSTHOOK: type: CREATETABLE
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@ice01
+PREHOOK: query: create external table ice02 (id int, key int) Stored by Iceberg stored as ORC
+    TBLPROPERTIES('format-version'='2')
```

Comment:
Nit: Looks like there is an additional tab spacing here. Can you please check and update the q.out file?

Reply:
Thanks for catching this! I changed the qtest but forgot to rerun it.  :(